db_waypoints/Waypoint[] waypoints
